plugins/nodes/src/js/columns/NodesTableRegionColumn.tsx

```diff
@@ -27,6 +33,5 @@ export function regionSorter(
   return sortDirection === "ASC" ? sortedData : sortedData.reverse();
 }
 export function regionSizer(args: WidthArgs): number {
-  // TODO: DCOS-38826
-  return Math.max(100, args.width / args.totalColumns);
+  return Math.max(170, args.width / args.totalColumns);
```

Comment:
Does the 170 magic number have some meaning?

Reply:
`min-width`

Reply:
Can we name it that then?